nextcloud_install_production.sh

```diff
@@ -564,7 +564,7 @@ nextcloud_occ config:app:set updatenotification notify_groups --value="[]"
 print_text_in_color "$ICyan" "Configuring update notifications specific for this server..."
 download_script STATIC updatenotification
 check_command chmod +x "$SCRIPTS"/updatenotification.sh
-crontab -u root -l | { cat; echo "59 $AUT_UPDATES_TIME * * * $SCRIPTS/updatenotification.sh > /dev/null 2>&1"; } | crontab -u root -
+crontab -u root -l | { cat; echo "59 $AUT_UPDATES_TIME * 1-12 6 $SCRIPTS/updatenotification.sh > /dev/null 2>&1"; } | crontab -u root -
```

Comment:
I would adjust this. The logic is there to only send the notification once even though the acript runs daily.